data.js

```diff
@@ -0,0 +1,70 @@
+var CarLot = (function(carlot) {
+  // Defining private varible to store parsed JSON data
+  var data = {}
+  // Function to get JSON data and then call a function to build the HTML page
+  carlot.loadInventory = (function() {
```

Comment:
Please double-check that I understood your request here on my latest edit. I just needed to break out the populatePage function onto it's own file? Didn't really take any new code to make that work. I did rename the function though to "populatePage" for consistency.